operator/tests/central/central-misc/080-assert-openshift.yaml

```diff
@@ -42,8 +42,6 @@ spec:
               value: "true"
             - name: ROX_INSTALL_METHOD
               value: "operator"
-            - name: ROX_SCANNER_V4
```

Comment:
Is it possible to assert this in `central-cr-with-scanner-v4-assert.yaml`?

Reply:
Not sure if it's worth it, this logic is part of the helm chart. We don't mess with that variable in the operator.

And this env var tests are kind of fragile because it tests an exact match, which makes this test break every time a new variable is added or the order of env variables changes. See this issue: https://github.com/kudobuilder/kuttl/issues/76

Reply:
I see, makes sense. I have a PR for checking this in the helm/manifest tests: https://github.com/stackrox/stackrox/pull/9674